src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoRecipeWrapper.java

```diff
@@ -307,11 +310,23 @@ public boolean handleClick(Minecraft minecraft, int mouseX, int mouseY, int mous
                     tooltip.set(k, TextFormatting.GRAY + tooltip.get(k));
                 }
             }
+            MetaTileEntity mte = GregTechAPI.META_TILE_ENTITY_REGISTRY.getObjectById(tooltipBlockStack.getItemDamage());
+            if(mte instanceof MetaTileEntityItemBus || mte instanceof MetaTileEntityFluidHatch) {
```

Comment:
It shouldn't be really checking for hatchets or buses, better check multiblock capability type instead